packages/block-library/src/image/edit.native.js

```diff
@@ -1,68 +1,112 @@
 /**
  * External dependencies
  */
-import { View, Image, TextInput } from 'react-native';
+import React from 'react';
+import { View, Image, TextInput, NativeEventEmitter } from 'react-native';
 import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
 
+const gutenbergBridgeEvents = new NativeEventEmitter( RNReactNativeGutenbergBridge );
+
 /**
  * Internal dependencies
  */
 import { MediaPlaceholder, RichText, BlockControls } from '@wordpress/editor';
-import { Toolbar, ToolbarButton } from '@wordpress/components';
+import { Toolbar, ToolbarButton, Spinner } from '@wordpress/components';
 import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import ImageSize from './image-size';
+import { isURL } from '@wordpress/url'
+
+const mediaUploadStateUploading = 1;
```

Comment:
Looking at other gutenberg code it looks constants have always defined all uppercase with the '_' separator between words.
That said if there is a way to define an Enum in JS it will be even better. 
I found this article that indicates a way: https://stijndewitt.com/2014/01/26/enums-in-javascript/

@Tug  any guidance for enums in JS?

Reply:
Enum isn't really a concept in JS, it being a weekly typed language, but as the article suggest there are equivalent patterns that could work just as well.

In our case, I think we should use const strings, just as we do for our redux action types. So:
```js
const MEDIA_UPLOADING_STATE_UPLOADING = 'MEDIA_UPLOADING_STATE_UPLOADING';
const MEDIA_UPLOADING_STATE_SUCCESS = 'MEDIA_UPLOADING_STATE_SUCCESS';
const MEDIA_UPLOADING_STATE_FAILURE = 'MEDIA_UPLOADING_STATE_FAILURE';
```
and bundle those into its own module so we can import those definitions from multiple places:
```js
import {
    MEDIA_UPLOADING_STATE_UPLOADING,
    MEDIA_UPLOADING_STATE_SUCCESS,
    MEDIA_UPLOADING_STATE_FAILURE
} from './media-upload-types';
```

Reply:
Oh right, the values, `1`, `2`, `3` are defined inside our native app. In that case, using the integer value is fine I think

Reply:
@SergioEstevao done. I have renamed so that we are following snake case with capital letters naming convention.